launcher/ui/MainWindow.cpp

```diff
@@ -1603,26 +1603,36 @@ void MainWindow::setCatBackground(bool enabled)
         QDateTime now = QDateTime::currentDateTime();
         QDateTime birthday(QDate(now.date().year(), 11, 30), QTime(0, 0));
         QDateTime xmas(QDate(now.date().year(), 12, 25), QTime(0, 0));
+
         QString cat;
+        if(APPLICATION->settings()->get("CatStyle").toString() == "BackgroundCat")
+        {
+            cat = "default";
+        }
+        else if(APPLICATION->settings()->get("CatStyle").toString() == "Jinx")
+        {
+            cat = "jinx";
+        }
+
         if(non_stupid_abs(now.daysTo(xmas)) <= 4) {
-            cat = "catmas";
+            cat += "Catmas";
         }
         else if (non_stupid_abs(now.daysTo(birthday)) <= 12) {
-            cat = "cattiversary";
+            cat += "Cattiversary";
         }
         else {
-            cat = "kitteh";
+            cat += "Cat";
         }
         view->setStyleSheet(QString(R"(
-InstanceView
-{
-    background-image: url(:/backgrounds/%1);
-    background-attachment: fixed;
-    background-clip: padding;
-    background-position: top right;
-    background-repeat: none;
-    background-color:palette(base);
-})").arg(cat));
+        InstanceView
```

Comment:
This was unindented for a reason - it's a raw string, any indentation you add will actually be in the string.

Let's focus on functionality changes now and if you want to, you can make a separate PR for formatting changes if you think it's necessary.